oxbow/src/gxf/format/gff.rs

```diff
@@ -0,0 +1,195 @@
+use std::io::{self, BufRead, Seek};
+
+use arrow::array::RecordBatchReader;
+use arrow::datatypes::Schema;
+use noodles::csi::binning_index;
+use noodles::csi::BinningIndex;
+
+use crate::gxf::batch_iterator::{BatchIterator, QueryBatchIterator};
+use crate::gxf::model::attribute::AttributeScanner;
+use crate::gxf::model::attribute::Push as _;
+use crate::gxf::model::field::DEFAULT_FIELD_NAMES;
+use crate::gxf::model::BatchBuilder;
+use crate::util::query::BgzfChunkReader;
+
+/// A GFF scanner.
+///
+/// # Examples
+///
+/// ```no_run
+/// use std::fs::File;
+///
+/// let inner = File::open("sample.gff")?;
+/// let mut fmt_reader = noodles::gff::io::Reader::new(inner);
+///
+/// let scanner = Scanner::new();
+/// let attr_defs = scanner.attribute_defs(&mut fmt_reader, Some(1000))?;
+/// ```
```

Comment:
The doctests are currently broken (even though they are `no_run`, they fail to compile with `cargo test`). Do you want to take a stab at fixing them? You can look at the other formats for a working pattern.